extensions/proposals/WEBGL_multisampled_render_to_texture/extension.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0"?>
+
+<extension href="WEBGL_multisampled_render_to_texture/">
```

Comment:
Please run "make" under the `extensions/` directory and fix the errors generated. At least this href is incorrect; there may be others.